Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM alpine:latest AS build
+
+# add required ca-certificates for https request.
+RUN apk --no-cache add ca-certificates
+
+# create empty directory for scratch image cache.
+RUN mkdir -p /tmp-syft
+
+# reduce container image to scratch size.
+FROM scratch
+
+# Copy directories and files needed to execute syft.
+COPY --from=build /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/ca-certificates.crt
+COPY --from=build /tmp-syft /tmp
```

Comment:
One option here would be to create the `/tmp` directory using `WORKDIR`. There's an argument to be made that this is a bit of a hack. It'd also simplify the Dockerfile and possibly remove the need for the multi-stage approach altogether. Example:

```Dockerfile
FROM scratch

WORKDIR /tmp  # used only to create this directory
WORKDIR /

COPY ./syft /

ENTRYPOINT [ "/syft" ]
```

Reply:
IMHO since the multi-stage approach is still required the `COPY` approach is OK (and is generally more explicit)

Reply:
Agreed